Makefile

```diff
@@ -94,9 +94,15 @@ clean:
 	rm -rf android/app/build
 
 post-install:
+	@cd ./node_modules/remotedev-server
+	@npm install socketcluster@5.0.4
+	@cd ../../
 	./node_modules/.bin/remotedev-debugger --hostname localhost --port 5678 --injectserver
 	@# Must remove the .babelrc for 0.42.0 to work correctly
-	rm -f node_modules/intl/.babelrc
+	# rm -f node_modules/intl/.babelrc remove
```

Comment:
This should be removed if it's not needed any more

Reply:
Will do